### PR TITLE
[torch] Add more functions to __init__.pyi.in for torch._C for Node and Value

### PR DESCRIPTION
**Pull request by dulinriley**
Summary:
https://github.com/pytorch/pytorch/pull/78757 recently added
a lot of functions to the type stub, but it missed a few of them.

This change will make sure every function is included, by making
sure this list is up-to-date with: `torch/csrc/jit/python/python_ir.cpp`.

This change only does this for Node and Value.

Differential Revision: D37189713

### CODE REVIEW

File: torch/_C/__init__.pyi.in [hunk 1]
```diff
@@ -456,8 +456,21 @@ class _InsertPoint:
...
+    def setDebugName(self, name: str) -> None: ...
+    def unique(self) -> _int: ...
+    def offset(self) -> _int: ...
+    def node(self) -> Node: ...
+    def uses(self) -> Iterator[Node]: ...
+    def replaceAllUsesWith(self, val: Value) -> None: ...
+    def replaceAllUsesAfterNodeWith(self, node: Node, val: Value) -> None: ...
     def requires_grad(self) -> _bool: ...
+    def requiresGrad(self) -> _bool: ...
+    def copyMetadata(self, from: Value) -> Value: ...
```
[the remark this answers is not in the document]
Comment (dulinriley) [reply torch/_C/__init__.pyi.in]: Thanks this was because `from` is a keyword and it can't be used as an argument name.
I fixed a similar issue with `is` as a function name later in this same file. Check the latest update.

**Review by ezyang — APPROVED** [review]
thanks

**Comment by dulinriley** [comment]
Adding `destroy` as a function too, then will merge

**Comment by dulinriley** [comment]
I don't have write access, so I can't apply a label. So I'll leave this in a comment instead:
topic: not user facing
